### PR TITLE
Update Fabric8 Kubernetes Client to 7.0.0

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Task

### Description

This PR updtes the Fabric8 Kubernetes Client to new major version 7.0.0.

### Checklist

- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -296,7 +296,7 @@
                 <exclusions>
                     <exclusion>
                         <groupId>io.fabric8</groupId>
-                        <artifactId>kubernetes-httpclient-okhttp</artifactId>
+                        <artifactId>kubernetes-httpclient-vertx</artifactId>
```
Comment (ppatierno) [inline pom.xml:299]: Out of curiosity ... does the fabric8 client have this Vert.x based http client to exclude?

Comment (scholzj) [reply pom.xml:299]: Fabric8 has been using for a long time OkHttp as the default HTTP client. In 7.0.0 it switched to using Vert.x Http Client by default. We however use already for a long time the JDK HTTP client. So we have to update the exclusion from the OkHttp to the Vertx HTTP client extension. The JDK HTTP client extension is included separately.

Comment (ppatierno) [reply pom.xml:299]: I did some research and realized that yes, they moved from okhttp to vertx as http client.

**Review by ppatierno — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (scholzj) [reply pom.xml:299]: (The Vrt.x HTTP Client is suported by Fabric8 already for some time, it just wasn't default so we didn't need to exclude it)

Comment (ppatierno) [reply pom.xml:299]: You wrote the comment when I had the older "version" of the page opened in the browser, so my last comment. :-)

**Pull request merged**